src/capitalsFirst.js

```diff
@@ -20,8 +20,24 @@
  *
  * @returns {string} - string with uppercase words in front
  */
+
 function capitalsFirst(str) {
-  // write code here
+  let bigFirstletter = [];
+  let smallFirsLetter = [];
```

Comment:
I wrote about let earlier.
You do not need let variable to do `array.push();`
Use let only if you want to reassign variables.
I once read this rule: "Always use `const`, use `let` only if you see an error \"Uncaught TypeError: Assignment to constant variable.\""

```
  const smallFirsLetter = [];
  smallFirsLetter.push('some value');

  smallFirsLetter = [1, 2, 3]; // throw error
```